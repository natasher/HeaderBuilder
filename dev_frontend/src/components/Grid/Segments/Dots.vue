<template functional>
  <span class="dots"></span>
</template>

<script>
export default {

  name: 'Dots',

}
</script>