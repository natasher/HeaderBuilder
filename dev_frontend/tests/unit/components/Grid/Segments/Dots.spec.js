import { shallowMount } from '@vue/test-utils'
import Dots from '../../../../../src/components/Grid/Segments/Dots.vue'

describe('Dots.vue', () => {

  test('should match snapshot', () => {
    const wrapper = shallowMount( Dots )

    expect( wrapper.element ).toMatchSnapshot()
  })

})