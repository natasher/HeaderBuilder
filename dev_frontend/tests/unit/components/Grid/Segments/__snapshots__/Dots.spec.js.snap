// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dots.vue should match snapshot 1`] = `
<span
  class="dots"
/>
`;
